governance.md

```diff
@@ -51,10 +51,26 @@ Subcommittees may be established by the Steering Council to assist in managing t
 
 ### Editorial policies and processes
 
-Individual journals are responsible for their own editorial process, including their editorial board composition. However, by being an Open Journals publication, each journal must:
+Individual journals are responsible for their own editorial process, including their editorial board composition. However, by being an Open Journals publication, each journal:
+
+**Must**
+
+- Be open access ([Diamond/Platinum or Gold](https://en.wikipedia.org/wiki/Open_access#Definitions)).
+- License journal content (e.g., articles) under a [CC-BY license](https://creativecommons.org/licenses/by/4.0/). 
+- Follow an open peer review process.
+- Publish a cost model (e.g., https://joss.theoj.org/about#costs).
```

Comment:
Do all our journals do this now (e.g. JOSE, JuliaCon, Open Journal of Astrophysics, Journal of Brief Ideas)

Reply:
I guess this actually goes for all the musts, not just this one...

Reply:
No. JOSE and JuliaCon are compliant I think, Open Journal of Astrophysics & Journal of Brief Ideas are partially compliant here.

I would suggest we address any low-hanging fruit here (i.e., add clear licensing if it doesn't exist) but otherwise grandfather these journals in, or simply don't 'count' them as Open Journals _journals_.

Reply:
I agree with @arfon